htdp-test/tests/htdp-lang/intm-lam.rktl

```diff
@@ -3,11 +3,6 @@
     exn:fail:contract?
     #rx"map: first argument must be a function that expects one argument"))
 
-(htdp-err/rt-test (map (lambda (x y) (+ x y)) (list 2 3 4))
-  (exn-type-and-msg
-    exn:fail:contract?
-    #rx"intm-lam.rktl"))
```

Comment:
This test has been moved to https://github.com/racket/drracket/pull/689/commits/4eaf30c4c74d0309d013c9332032d779e777d647.